Bottom: db812b75ecaeee8341e4b85d2e454e10685166e3
Top:    6f6862d36c9d30dd4dec1e2d2343ea07e91662d6
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-12-23 00:44:35 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
